ember-cli-build.js

```diff
@@ -300,7 +300,15 @@ module.exports = function(options) {
   });
 
   emberTestingBundle = concat(emberTestingBundle, {
-    outputFile: 'ember-testing.js'
+    outputFile: 'ember-testing.js',
+    hasBootstrap: false,
+    footer: stripIndent`
+      var testing = requireModule('ember-testing');
+      Ember.Test = testing.Test;
```

Comment:
Is `Ember` guaranteed to exist here? What happens when you eval `ember-testing.js` before `ember.debug.js` or `ember.prod.js`?